packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -38,6 +38,31 @@ Set the `size` prop to `"large"` to use the large Spinner.
   <Story id="core-spinner--large" />
 </Canvas>
 
+## Usage
```

Comment:
We should add examples that follow the usage guidance specified on this page cc @libertymayc

Reply:
I see you've created the examples, but they haven't been added to the docs? Also, the guidance on when to use large or default spinners seems very hidden cc @sion-docs @mikearildbrown

Reply:
thank you, updated